qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +68,36 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat': IntSequenceDirectoryFormat,
+            'MappingDirectoryFormat': MappingDirectoryFormat,
+            'IntSequenceV2DirectoryFormat': IntSequenceV2DirectoryFormat,
+            'IntSequenceFormatV2': IntSequenceFormatV2,
+            'FourIntsDirectoryFormat': FourIntsDirectoryFormat,
+            'IntSequenceFormat': IntSequenceFormat
+        }
+        self.assertEqual(set(obs), set(exp))
+
+    def test_importable_formats_with_dummy(self):
+        obs = self.pm.importable_formats
+        self.assertFalse('UnimportableFormat' in obs.keys())
+        self.assertFalse(UnimportableFormat in obs.values())
```

Comment:
This will never evaluate `True` because `.values()` contains `FormatRecords` instead of the formats themselves. I'm okay with removing this assertion (and the corresponding directory format assertion below) and just check the keys like you're doing.